scripts/bootstrap.sh

```diff
@@ -34,6 +38,8 @@ while getopts "m:n:a:s:o:" flag; do
   a) NUM_AUTHORITIES=${OPTARG} ;;
   s) SECRET=${OPTARG} ;;
   o) OUTPUT_DIR=${OPTARG} ;;
+  i) NETWORK_ID=${OPTARG} ;;
```

Comment:
Also add `:i:c` as a flag above in the while statement.

Reply:
Done